azure_ad.sh

```diff
@@ -94,7 +94,7 @@ function addEnvToFile {
 function setVariableFile {
   local __filePath="${1}"
   echo $__filePath
-  if [[ -f ${__filePath} ]]; then
+  if [[ -f ${__filePath} ]]  && [[ "$(grep -v '^\s*$\|^\s*\#' ${__filePath})" != "" ]]; then
```

Comment:
what does the new condition do ? Maybe adding a comment above this line briefly documenting what it does would be a good idea in general ... or perhaps extracting the if into a function, where the name provides enough detail about what it does (if possible/realistic in bash)

Reply:
Comment added to all places where this condition is used. 
Extraction to another function is not necessary because it is already extracted to the function with a self-describing name **setVariableFile / unsetVariableFile**

Added comment: **# check if the file exists and if contain at least one config parameter**